packages/block-library/src/navigation/edit/index.js

```diff
@@ -339,6 +357,7 @@ function Navigation( {
 	// Turn on contrast checker for web only since it's not supported on mobile yet.
 	const enableContrastChecking = Platform.OS === 'web';
 
+	const [ convertingClassicMenu, setConvertingClassicMenu ] = useState();
```

Comment:
See previous comment about already having [this state available from the hoo](https://github.com/WordPress/gutenberg/blob/349240ac3ea4890f0a95b07e4d089b4307a6eab7/packages/block-library/src/navigation/edit/use-convert-classic-menu-to-block-menu.js#L149)k.

Reply:
I've removed this state and used the existing state in https://github.com/WordPress/gutenberg/pull/44173/commits/1c8f006cc099283a6f32875f38948f9a9685aa74